### DISCUSSION TITLE
FBA Inbound v2024 placement / transport options

### DISCUSSION THREAD

**Discussion by NoxArt** [amzn/selling-partner-api-models#4386] [Q&A] [answered]
Hi,
* in V0 we were using NonPartneredSmallParcelDataInput type=SP carrier=Other
* in v2024 I createInboundPlan with EU marketplace, then generatePlacementOptions with empty request
* then the list of placementOptions is only 1 option
* we confirm it
* then in generateTransportOptions -> listOfTransportationOptions there are all only FREIGHT_LTL and no GROUND_SMALL_PARCEL + also 99% of offered carriers are from China ... this seems wrong? I have no idea what I'm doing wrong

Thank you

**Comment by NoxArt** [comment]
also using new FreightInformation(freightClass: "NONE") did not help

**Comment by NoxArt** [comment, marked as answer]
It looks like we're wrong went the B branch of the workplace where we should have gone A (left side)

This way we got GSP

**Comment by puppsupr** [comment]
Closing this as its answered in #4387.
